src/Visualization/Python/InterpolateVolumeData.py

```diff
@@ -143,6 +147,8 @@ def interpolate_h5_file(source_file_path,
             for name in tensor_names
         ]
 
+        source_file.close()
+
```

Comment:
I found I need to close `target_file` in line 179 before `get_vol` is called for this to work.